features/bevragen/autorisatie-gba.feature

```diff
@@ -0,0 +1,590 @@
+      | 010120 020210 020220 020230 020240 | niet geautoriseerd voor ouder 2       |
+      | 010120 030210 030220 030230 030240 | niet geautoriseerd voor ouder 1       |
+
+    Abstract Scenario: Afnemer is wel geautoriseerd voor ten minste één gegeven van ouder 1 en van ouder 2
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | <ad hoc rubrieken>              | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode |        |
+      En de persoon met burgerservicenummer '000000024' heeft een ouder '1' met de volgende gegevens
+      | voornamen (02.10) |
+      | Chantal           |
+      En de persoon heeft een ouder '2' met de volgende gegevens
+      | voornamen (02.10) |
+      | Mark              |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000024                       |
+      | fields              | ouders.ouderAanduiding          |
+      Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
+      | naam            | waarde  |
+      | ouderAanduiding | 1       |
+      En heeft de persoon een 'ouder' met de volgende gegevens
+      | naam            | waarde |
+      | ouderAanduiding | 2      |
+
+      Voorbeelden:
+      | ad hoc rubrieken                   | autorisatie                                            |
+      | 010120 020210 030210               | exact 1 gegeven van ouder 1 en 1 gegeven van ouder 2   |
+      | 010120 020120 020210 030120 030210 | meerdere gegevens van ouder 1 en 1 gegeven van ouder 2 |
+  
+  
+  Rule: Een gemeente als afnemer is geautoriseerd voor alle gegevens van eigen inwoners
+
+    Scenario: Gemeente raadpleegt een eigen inwoner en fields vraagt om partners.geslacht (50410) dat niet in de autorisatie zit
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 010120 050240 050610            | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode | 0599   |
+      En de persoon met burgerservicenummer '000000024' heeft de volgende 'verblijfplaats' gegevens
+      | gemeente van inschrijving (09.10) |
+      | 0599                              |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | gemeente_code | 0599 |
```

Comment:
Is ook de gemeente_code nodig om te bepalen dat de betreffende persoon een inwoner is van de gemeente? Is de gemeente van inschrijving niet voldoende?